### PR TITLE
Change button lineheight to 1.1rem

### PR DESCRIPTION
**Pull request by djnunez-aot**
*Issue #1285 :*
https://github.com/bcgov/met-public/issues/1285

-Update button line height

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the met-public license (Apache 2.0).

### CODE REVIEW

File: met-web/src/styles/Theme.ts [hunk 1]
```diff
@@ -62,6 +62,7 @@ export const BaseTheme = createTheme({
         MuiButton: {
             styleOverrides: {
                 root: {
+                    lineHeight: '1.1rem',
```
Comment (jadmsaadaot) [inline met-web/src/styles/Theme.ts:65]: can we remove the style overrides here and just add them to PrimaryButton and SecondaryButton

**Review by jadmsaadaot — CHANGES_REQUESTED** [review]
Move overrides to common atomic components

File: met-web/src/styles/Theme.ts (hunk 1 above)
Comment (djnunez-aot) [reply met-web/src/styles/Theme.ts:65]: sure thing

Comment (jadmsaadaot) [reply met-web/src/styles/Theme.ts]: have we also removed any other style overrides here and moved them to the atomic components?

**Review by saravanpa-aot — APPROVED** [review]

**Pull request merged**